.gitignore

```diff
@@ -16,6 +16,7 @@
 npm-debug.log*
 yarn-error.log
 testem.log
+.vscode
```

Comment:
for now I've added it to gitignore,
If you think it's worth it - I could commit some vscode config that is compatible with this repo

Reply:
That's fine for now, thank you :)